apiserver/README.md

```diff
@@ -76,35 +115,34 @@ kubectl get pods
 
 ## Usage
 
-After the deployment we may use the `{{baseUrl}}` to access the
+After the deployment we may use the `{{baseUrl}}` to access the service. See [swagger support section](https://ray-project.github.io/kuberay/components/apiserver/#swagger-support) to get the complete definitions of APIs.
 
-- (default) for nodeport access, we provide the default http port `31888` for connection and you can connect it using.
+* (default) for nodeport access, use port `31888` for connection
 
-- for ingress access, you will need to create your own ingress
+* for ingress access, you will need to create your own ingress
 
```

Comment:
You can have a helm chart to create an ingress or route, by modifying values.yaml

Reply:
Perhaps we can add this in a later commit.